Patch already in upstream master:

JENT: use proper pointer dereferencing

Signed-off-by: Stephan Mueller <smueller@chronox.de>

---

diff -aur a/jitterentropy-base-user.h b/jitterentropy-base-user.h
--- a/jitterentropy-base-user.h
+++ b/jitterentropy-base-user.h
@@ -290,12 +290,12 @@
 		ext = strstr(buf, "K");
 		if (ext) {
 			shift = 10;
-			ext = '\0';
+			*ext = '\0';
 		} else {
 			ext = strstr(buf, "M");
 			if (ext) {
 				shift = 20;
-				ext = '\0';
+				*ext = '\0';
 			}
 		}
 
